x-pack/plugins/cloud_security_posture/public/pages/rules/rules_bottom_bar.tsx

```diff
@@ -16,7 +16,15 @@ interface RulesBottomBarProps {
 }
 
 export const RulesBottomBar = ({ onSave, onCancel, isLoading }: RulesBottomBarProps) => (
-  <EuiBottomBar>
+  <EuiBottomBar
+    style={{
+      // same as .euiFlyout
+      zIndex: 1000,
```

Comment:
Don't we want this to always override the flyout?

Reply:
in second thought yeah, i think this only works because of the ordering of DOM elements

thanks! will increment it